test_case_gen.py

```diff
@@ -0,0 +1,120 @@
+"""
+Generates Test Cases 0, 1 and saves them to json

```

Comment:
You should mention that these test cases are generated for each kind of Flow (FlowDummy, FlowReno, FlowFast).

You should also put all of the files in a data/ folder (where the "data" folder is in the same directory as this script). This should be mentioned here. Also, the format of the file names should be mentioned, i.e. "test_case_<num>_<flow_type>.json where num is ... and flow_types is ..."